sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -73,12 +77,17 @@ export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
   }
 
   /**
-   * Get all COE denied reasons, which are active.
-   * @returns COE denied reason list.
+   * List of possible COE denied reasons.
+   * @param offeringIntensity Used to filter the denied reasons based on offering intensity.
```

Comment:
Offering intensity to filter the denied reasons?

Reply:
I did not follow. Is it a suggestion to a clear parameter description, as below?

> @param offeringIntensity Offering intensity to filter the denied reasons.